apps/frontend/src/pages/management/libs/components/groups-tab/libs/components/edit-group-modal/edit-group-modal.tsx

```diff
@@ -0,0 +1,105 @@
+import { Button, Modal } from "~/libs/components/components.js";
+import { PermissionKey } from "~/libs/enums/enums.js";
+import { useAppDispatch, useCallback } from "~/libs/hooks/hooks.js";
+import { type UserAuthResponseDto } from "~/modules/auth/auth.js";
+import {
+	type GroupResponseDto,
+	actions as groupsActions,
+} from "~/modules/groups/groups.js";
+import { type PermissionResponseDto } from "~/modules/permissions/permissions.js";
+import { EditCheckbox } from "~/pages/management/libs/components/edit-checkbox/edit-checkbox.jsx";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	group: GroupResponseDto | null;
```

Comment:
```suggestion
	group: GroupResponseDto;
```
it can not be null since this popup is opened only when you have a `group` in state, is not it?

Reply:
@what1s1ove 
After adding animation for popups, the popup renders once when the page loads, not every time when the popup opens. To not pass a null value to custom popups, I need to use conditional rendering. But it will disable popup animation.
What should I leave in this case? Popup animation or passing not null values?

Reply:
both. try to find a way to fix it

Reply:
@what1s1ove 
I tried different ways to keep an animation and not to pass null values, but as a result, I received errors of reading null values or non-animated popup. I asked for help from my teammate, but the result was the same.
One idea that works - it's to pass a some default user or group to popup if the current user is null. But, I think, it's a bad solution, isn't it?
I pushed this solution, so I will wait for your answer.